CHANGELOG.md

```diff
@@ -63,6 +63,13 @@ Please see the [Envoy documentation](https://www.envoyproxy.io/docs/envoy/latest
 
 ## RELEASE NOTES
 
+## Next Release
```

Comment:
`make VERSION=1.13.10 release/hotfix/start` or the release process will be all jammed up for the next person :)

Reply:
Thanks for the catch!